src/app/shared/services/blockchain/flavors.ts

```diff
@@ -1,4 +1,4 @@
 export type IssuerFlavor = 'IssuerV1'
-export type AssetFlavor = 'AssetV1' | 'AssetTransferableV1'
-export type CampaignFlavor = 'CfManagerSoftcapV1'
+export type AssetFlavor = 'AssetV1' | 'AssetTransferableV1' | 'AssetSimpleV1'
```

Comment:
You can do this:
```typescript
export const AssetFlavors = ['AssetV1', 'AssetTransferableV1', 'AssetSimpleV1'] as const
export type AssetFlavor = typeof AssetFlavors[number]
```
That way, you will already get a constant array of all asset flavors which you can use across the app. The added bonus here is that `AssetFlavor` type resolves to `'AssetV1' | 'AssetTransferableV1' | 'AssetSimpleV1'` - so when you add a new value to the array it will also update your type as well!